Formulaic TOS

```diff
@@ -0,0 +1,73 @@
+#Terms of Service - Mozilla Formulaic
+
+Last Updated March 8, 2024
+{: datetime="2024-03-08" 
+
+Formulaic (the “Service”) is a service that allows users to create, share, manage, and use templates (the “Formulas”) for querying Artificial Intelligence (AI) based Large Language Model (LLMs) (the “Service”).  It also allows users to run those queries through a third party LLM platform, and to view the resulting responses. (For the purposes of this Agreement, Formulas, instructions, descriptions, and responses will all be referred to as the “Content”.) The Service is owned, licensed, and operated by Mozilla Corporation (referred to herein as "we," "us," "our," or "Mozilla"). 
+Please read these terms of service (“Agreement” or “Terms”) carefully before you sign up for or use the Service.  The Terms constitute a binding legal agreement, which govern your use of the Service via any platform or device.  By accessing, signing up for, or using the Service, you agree to be bound by these Terms.  
+
+##1. The Service
+The Service (including both the platform and the Formulas shared by other users, as well as your ability to submit and/or run the Formulas using third party LLMs) is provided solely for use according to these Terms.  If you agree to these terms on behalf of a third party, such as your employer, you represent that you are authorized to, and intend to bind that party to these Terms.
+By accepting these terms, you also accept the Formulaic Privacy Notice and Formulaic Content Policies.
+
+###A. Creating an Account
+In order to view or run shared queries, or to share content of your own, you will need to create an account on the Service using your Google Account.  You can create a Google Account on [Google’s signup page](https://accounts.google.com/lifecycle/steps/signup/name?dsh=S1694799353:1707861924262237&flowEntry=SignUp&flowName=GlifWebSignIn&theme=glif&TL=ADg0xR0JpndiwBwSSLq5Ih_cBj1vPyVBRu2lVXVKxOqIHKvLx__ThXaRX7v0OVnt).  More information about the information that we receive from Google, and how we use that information, is included in the Formulaic Privacy Notice. Google Accounts has it’s own [Privacy Notice](https://policies.google.com/privacy?hl=en-US).
```

Comment:
The link seems to include way too many parameters. Pointing to `https://accounts.google.com/` should be enough?

Reply:
Yes, that works.  Thanks